app/src/components/Scheduling/Prompt.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { logo, Typography, Button } from '../../store/index';
+import { useStyles } from './PromptStyles';
+
+export default function Prompt() {
+  const classes = useStyles();
+  return (
+    <div className={classes.root}>
+      <img src={logo.default} className={classes.img} alt='logo' />
+      <Typography
+        variant='h4'
+        color='primary'
+        style={{
+          fontSize: '32px',
```

Comment:
I tried to follow the exact styles based on Figma, but didn't find any matching styles that have the same font size or font weight from `theme.js`. Should I find a variant in `theme.js` that closely resembles the style in Figma?

Reply:
OK good catch. I think there's a mistake in `theme.js`. h4 font size should be 32px based on the style guide: https://www.figma.com/file/kwUcXHYpKMHhuktdElLmmw/Calendaring-Home-Unite-Us-(Copy)?node-id=885%3A1232

Would you mind changing the theme object? This may screw up stuff in other places, but I think it's important enough to change it now, and I'll go through this weekend and make sure everything looks ok.